monailabel/scripts/monailabel

```diff
@@ -17,8 +17,8 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd)"
 export PYTHONPATH=$DIR:$PYTHONPATH
 echo Using PYTHONPATH=$PYTHONPATH
 
-PYEXE=python
-version=$(python --version 2>&1)
+PYEXE=${MONILABEL_PYEXE:-python}
```

Comment:
There is a typo here `MONILABEL_PYEXE` should be probably `MONAILABEL_PYEXE`.

Reply:
Thanks, @lassoan
Fixed this here https://github.com/Project-MONAI/MONAILabel/pull/894